js/js.translator/testData/box/jsModule/externalConstructor.kt

```diff
@@ -0,0 +1,24 @@
+// EXPECTED_REACHABLE_NODES: 1334
+// MODULE_KIND: AMD
+package foo
+
+@JsModule("lib")
```

Comment:
Isn't it reproduced without any module systems?

Reply:
Works in both cases.
Modular variant can be simply tested in [separate project](https://github.com/turansky/kotlin-battleground/blob/master/KT-31126/src/test/kotlin/foo/externalConstructor.kt)

Reply:
My thought was that (without the fix) it should be reproducible without any module system. Isn't it?

Reply:
Yes, it is.

Reply:
Modular system used like more typical.